components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -2544,27 +2555,41 @@ public static void addDefinedAllSequencesToRegistry(UserRegistry registry,
 					"Error while saving defined sequences to the registry ", e);
 		} catch (IOException e) {
 			throw new APIManagementException("Error while reading defined sequence ", e);
+		} finally {
+			if (inSeqStream != null) {
+				try {
+					inSeqStream.close();
+				} catch (IOException e) {
+					log.error(
+							"Error while closing input stream in path " + seqFolderLocation + " " +
+							e);
+				}
+			}
 		}
 
 	}
 
+	public static void writeDefinedSequencesToTenantRegistry(int tenantID)
+			throws APIManagementException {
```

Comment:
Fixed in #1581 (https://github.com/wso2/carbon-apimgt/commit/9806881b73d7497356dcd453fa950b63c644f2c4)